### ISSUE TITLE
Synchronization Domains

### ISSUE THREAD

**Issue by reteprelief** [saeaadl/aadlv3#9]
Can we improve the V2 specification? 
Relevant properties
Relation of synchronous system annex.

**Comment by yoogx** [comment]
Agreed this is an important point, it will be required to support TSN and similar network protocols, as we need to quantify and check for specific clock drifts.
